consensus/XDPoS/engines/engine_v2/engine.go

```diff
@@ -163,7 +169,7 @@ func (x *XDPoS_v2) initial(chain consensus.ChainReader, header *types.Header) er
 	var quorumCert *types.QuorumCert
 	var err error
 
-	if header.Number.Int64() == x.config.V2.FirstSwitchBlock.Int64() {
+	if header.Number.Int64() == x.config.V2.SwitchBlock.Int64() {
```

Comment:
I'm getting confused, why does v2 still have switchBlock? shouldn't we just use switchRound only?

Reply:
or, this SwitchBlock is actually the bigger v2 switch. i.e v1 to v2 switch

Reply:
Yes, this is the switch block between v1 and v2